opflexagent/gbp_ovs_agent.py

```diff
@@ -35,7 +35,6 @@
 from opflexagent import as_metadata_manager
 from opflexagent import constants as ofcst
 from opflexagent import rpc
-from opflexagent import snat_iptables_manager
```

Comment:
don't remove this line. SNAT stuff seems a little broken on 'master', it is missing this line: https://github.com/noironetworks/python-opflex-agent/blob/stable/juno/opflexagent/gbp_ovs_agent.py#L231